easybuild/framework/easyblock.py

```diff
@@ -3416,6 +3424,24 @@ def build_and_install_one(ecdict, init_env):
     return (success, application_log, errormsg)
 
 
+def copy_easyblocks_for_reprod(easyblock_instances, reprod_dir):
+    reprod_easyblock_dir = os.path.join(reprod_dir, 'easyblocks')
+    easyblock_paths = set()
+    for easyblock_instance in easyblock_instances:
+        for easyblock_class in inspect.getmro(type(easyblock_instance)):
+            easyblock_path = inspect.getsourcefile(easyblock_class)
+            # if we reach EasyBlock or ExtensionEasyBlock class, we are done
+            # (ExtensionEasyblock is hardcoded to avoid a cyclical import)
+            if easyblock_class.__name__ in [EasyBlock.__name__, 'ExtensionEasyBlock']:
```

Comment:
@ocaisa To be on the safe side, we also should consider `'Extension'` here... In theory there could be extensions that are installed with a class that directly derives from `Extension` rather than `ExtensionEasyBlock` (we don't have any like these currently in the central easyblocks repository though....)

Reply:
@boegel This comes at no cost right, I should do this